### PR TITLE
opencv: revision to fix broken bottle

### PR DESCRIPTION
**Pull request by commitay**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
opportunistically linked to tesseract and leptonica

disable hdf5, szip, tesseract and leptonica

### CODE REVIEW

**Comment by fxcoudert** [comment]
```
failed: brew linkage --test visp
        Broken dependencies:
  /usr/local/opt/opencv/lib/libopencv_text.3.4.dylib (opencv)
```

File: Formula/opencv.rb [hunk 1]
```diff
@@ -57,7 +57,9 @@ def install
       -DBUILD_TESTS=OFF
       -DBUILD_TIFF=OFF
       -DBUILD_ZLIB=OFF
+      -DBUILD_opencv_hdf=OFF
       -DBUILD_opencv_java=OFF
+      -DBUILD_opencv_text=OFF
```
Comment (legraphista) [inline Formula/opencv.rb]: Hey @commitay!
I believe this line is a bit problematic here. Just spent the last 30 minutes trying to figure out why some of my projects which depend on opencv suddenly stopped working after a `brew update & brew upgrade` with the following error: `Library not loaded: /usr/local/opt/opencv/lib/libopencv_text.3.4.dylib`

Comment (ilovezfs) [reply Formula/opencv.rb]: Did you try rebuilding them? Linkage to `libopencv_text.3.4.dylib` should not be needed following a rebuild of the dependent project.

Comment (legraphista) [reply Formula/opencv.rb]: Ah, yes! You are right, a rebuild will do the trick.

**Pull request closed without merging**